README.md

```diff
@@ -1,5 +1,5 @@
 # fog-packet
-Packet provider for Fog
+WIP: Packet provider for Fog
```

Comment:
Some explanatory text at the top would be helpful.

Fog is ...
Packet is ..
The `fog-packet` package allows you to ...

For help with this package:

* Open up an issue in this repository
* Contact the Packet Community Slack via http://slack.packet.net or on Freenode IRC on the #packethost channel.
* Search the Packet knowledge base at http://help.packet.net